libbeat/common/kubernetes/metadata/pod.go

```diff
@@ -99,6 +98,17 @@ func (p *pod) GenerateK8s(obj kubernetes.Resource, opts ...FieldOptions) common.
 		}
 	}
 
+	// check if Pod is handled by a Job which is controlled by a CronJob
```

Comment:
Why is this commented out?

Reply:
I was just checking the behaviour of CI.